_posts/2016-10-27-journal-of-a-junior-multisite.md

```diff
@@ -0,0 +1,133 @@
+
+
+###Step 1: Setting up
+
+For those who haven't used Umbraco before, you can get your installation [here](https://our.umbraco.org/documentation/Getting-Started/Setup/Install/).
+
+
+I'm going to be showing the multisite we made for [HRGO Recruitment](http://www.hrgo.co.uk/) in this example. You will want to start with creating some templates for your sites. I personally have used the same templates for each different site to save a huge library of unique templates. I prefer having reusable templates and this is very easy with the use of the [Umbraco Grid](https://our.umbraco.org/documentation/getting-started/backoffice/property-editors/built-in-property-editors/grid-layout) and special [grid editors](https://our.umbraco.org/documentation/getting-started/backoffice/property-editors/built-in-property-editors/grid-layout/grid-editors). I won't go into this too much, as there are many resources on the web for this.
+
+
+You will want a 'Homepage' document type, as well as a 'Website Settings'. These are a must - the homepage will be the start node, and in the website settings, we will set each websites unique stylesheet and settings. This isn't necessary but is good to know if each site is going to be different style wise.
+
+
+In my Homepage I have the Umbraco Grid:
+
+![Homepage Settings][homepage]
+
+[homepage]: /assets/images/blog/ums-homepage-settings.png "Homepage Settings"
+
+In my Website Settings, I have some text fields to enter the name or your stylesheet and also fonts.
+
+![Website Settings][website]
+
+[website]: /assets/images/blog/ums-website-settings.png "Website Settings"
+
+###Step 2: Allowing each site to have a unique domain
+
+
+Once you've got Umbraco installed, you will want to go into umbracoSettings.config, and find '<requestHandler>' and set <useDomainPrefixes> to true. This allows you to have unique domains for multiple sites.
+
+![Request Handler][request]
+
+[request]: /assets/images/blog/ums-request-handler.png "Request Handler"
+
+###Step 3: Creating content for each site
+
+
+Now these have been made, we will create them in the Content tab. Remember to set up permissions to allow the Website Settings document to be a child of the Homepage, along with any other Doctypes you have made. It should look a little like the image below:
+
+![Content Tree][content]
+
+[content]: /assets/images/blog/ums-content-tree.png "Content Tree"
+
+
+Now give each site its own unique content! If each site is having its own stylesheet or fonts, then this is where you will add them into the settings. First, you will want to make your stylesheets have been created. In the settings, you can add the file name and your link to the font you have chosen. I'm using google fonts, so I can just add the google link but this can easily be customised to suit you.
+
+
+We used a helper class to get the website settings node:
+<xmp>
+public static IPublishedContent GetWebsiteSettings()
+{
```

Comment:
btw idk if we should present this piece of code... I think it's suboptimal in terms of caching